app/models/file_search.rb

```diff
@@ -80,16 +80,17 @@ def files
     params.empty? ? highlighted_files : filtered_files
   end
 
-  private
-
-  VENUES = ["Elizabethan", "Angus Bowmer", "Thomas", "The Green Show"]
+  PRIMARY_VENUES = ["Elizabethan", "Angus Bowmer", "Thomas", "The Green Show"]
```

Comment:
They are essentially private, but used by the tests, so we put them here on purpose.